modules/making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,6 @@
+[preface]
+
+[id="making-open-source-more-inclusive"]
```

Comment:
```suggestion
[id="making-open-source-more-inclusive_{context}"]
```

Reply:
@ncbaratta Not sure if adding `{context}` would be good for other runtimes docs or not. If the topic isn't in an assembly that provides a context it results in a build failure.